solidstart-tanstackquery-tailwind-modules/src/components/RepoAbout/Description.tsx

```diff
@@ -0,0 +1,20 @@
+import { Match, Switch } from 'solid-js';
+
+interface DescriptionProps {
+  text: string;
```

Comment:
Inside the render, we check if this prop could be undefined, so probably here we must type is as `text?:string`